### PR TITLE
Fixes stun revolvers not going on security belts

### LINKED ISSUE #21278: Secbelt cannot hold stun revolvers
(WEB REPORT BY: chazach REMOTE: 206.221.180.138:7777)
# Revision
0e50b00c6c00b3f683b4b76885e5073d2b636989

# Description
just seems like an oversight since they are meant to be the "upgrade" to tasers

### PR DESCRIPTION
**Pull request by ShiftyRail**
Fixes #21278

### CODE REVIEW

**Comment by iFlashYou** [comment]
>security buffs

**Comment by SonixApache** [comment]
discussed many times in the past 5 years, passed on many times in the past 5 years

**Comment by ShiftyRail** [comment]
Is it really that big of a deal ?

**Comment by SonixApache** [comment]
it's tasercreep

**Comment by DrSnips** [comment]
Why not just make stun revolvers a child of tasers?

**Comment by Chazach** [comment]
@SonixApache I am not that invested in this but when was it discussed and turned down many times before?

**Comment by SonixApache** [comment]
In the thread, in the IRC, in the discord...

Stun revolvers are essentially "taser cannons" and it's fine that they only fit on your armor imo.

**Comment by DrSnips** [comment]
[citation needed]
A PR number, discord screen shots, or thread links maybe?

**Comment by SonixApache** [comment]
the only discussion i can find with my limited amount of caring is one where i myself asked if we should add it to the belts, and i got one yes and two "wait that's a thing?"s so whatever

still think its seccreep and bad

**Comment by DrSnips** [comment]
Coming up next: Local anon asks "How many shots does the stun revolver actually have compared to the normal taser?"

**Comment by ghost** [comment]
The only difference between a taser and a stunrevolver is that a stun revolver can fire 3 more rounds.
5 vs 8

**Comment by SonixApache** [comment]
62% increase of munitions per gun, balanced by the fact that they only fit on your armor or in your pack

**Comment by DrSnips** [comment]
I hate the meme that upgrades can't exist without some kind of drawback. Balance is for when two+ items are immediately available at the same time and there's trade-offs to using them. An example of this is tasers vs. beanbag/stun shotguns.

**Comment by ghost** [comment]
I personally don't understand why straight up upgrades can't exist.

**Comment by SonixApache** [comment]
On the contrary, I dislike the idea that there should be "direct upgrades" to gear that doesn't really need upgrading, especially when doing so would only make it harder for interesting things to happen.

**Comment by DrSnips** [comment]
>interesting things to happen

This point is subjective. Interesting for who? Sec or whoever is on the other end? As a sec officer I would say its more interesting for me to have 8 shots allowing for me to handle more/bigger situations. As a target I could also say its more interesting since I  would need to change my game plan to account for more shots based on the sound of the thing shooting at me.

**Comment by SonixApache** [comment]
You can make the decision to carry more shots on you in your limited carry space. Reolver on the hip, or revolvers in your pack, or a shotgun on the hip and a belt full of shells. There's a lot possible as-is, which is what makes the different options interesting.

Changing the amount of easily-accessible nobody-will-bitch-at-you-for-it firepower people can carry only makes things harder for antags (and non-antags) that are trying something interesting to not get instant-fucked, and makes it yet another one of those checkmark things you're supposed to get as a round progresses, like the current state of medbay.

**Comment by SonixApache** [comment]
also 🆑

**Comment by Accipitermaris** [comment]
>SonixApache
>In the thread, in the IRC, in the discord...
> 
> Stun revolvers are essentially "taser cannons" and it's fine that they only fit on your armor imo.

Since Energy Guns and Stun Revolvers have the same size restrictions (Can't fit in a Security Belt, but can fit on armor and in a holster), and since Energy Guns (10 stuns) have more taser shots than Stun Revolvers (8 stuns), they are entirely superior to Stun Revolvers. ("If you get shoved, they can use it against you" is highly applicable to any firearm that can stun or kill, and pretty much only stops being highly applicable once we've downgraded all the way to the topic of hand-held flashes. In the hands of someone with intent to kill, a stunned cop is a dead cop.)

TL;DR: eguns are superior to stun revolvers as-is because they fill the same inventory niche and eguns entirely outclass them in that niche. this pr would change that. depending on your view that's good or bad, yadda yadda. but i think the first point is fairly uncontestable

**Comment by SonixApache** [comment]
Arming people with eguns on code green is a big no-no.
Eguns, as opposed to tasers/stunvolvers, are very prone to being used for murder, which makes distributing them to people a liability rather than a good tool for arresting / stun & run defense.
Saying they are flatout "better" is very subjective. Good comparison, though, does show the reference.

What if this change was bundled with turning stun revolvers into modkits for tasers, much like AEG kits for eguns?

**Comment by DrSnips** [comment]
Id be for them being kits if it was just to make them children of tasers(meaning they then fit in belts) but a word of warning. Im dealing with the upset protolathe people in the ammolathe PR where they don't quite like how designs are slowly getting removed. I also don't think making them kits does anything except affect science when you could just flat out do this PR or make them children of tasers without dealing with a kit.

**Comment by Accipitermaris** [comment]
>Arming people with eguns on code green is a big no-no.

codes dont say anything about lethality of weapons permitted, only where weapons are permitted to be stored. if theres a reason to arm them, theres a reason to arm them. if there isnt, then there isnt. 

never mentioned handing out either weapons to randos but a laser gun with no capacity for stunning is safer for sec than both an egun and a basic taser. in that respect i dont see where giving stun revolvers or eguns to randos enters the equation

**Review by MadmanMartian — APPROVED** [review]
there has to be a better way of populating the can_only_hold
